### PR TITLE
Added a general search engine

### PR DESCRIPTION
**Pull request by biswabaibhab007**
Added a general search engine
 - AOL search

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [DuckDuckGo](https://duckduckgo.com/)
 - [Swisscows](https://swisscows.com/en)
 - [Naver](https://www.naver.com/)
+- [Ecosia](https://www.ecosia.org)
```
Comment (edoardottt) [inline README.md:48]: ecosia is just a web portal using bing, we can remove it

File: README.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
...
+- [AOL Search](https://search.aol.com)
```
Comment (edoardottt) [inline README.md:49]: Here `AOL` as title is okay, remove `search`

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (biswabaibhab007) [reply README.md:48]: Done

Comment (edoardottt) [reply README.md:48]: it's not

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (biswabaibhab007) [reply README.md]: sorry my bad, I think everything is done now.

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**